### PR TITLE
Can change values before starting wappsto

### PR DESCRIPTION
**Pull request by bubriks**
Changing of initial value and other is possible before calling start at Wappsto.

[#19 ](https://github.com/Wappsto/wappsto-python/issues/19)

### CODE REVIEW

File: wappsto/object_instantiation/instantiate.py [hunk 1]
```diff
@@ -258,3 +258,99 @@ def build_network(self, decoded):
...
+                    state = {
+                        'data': value_iterator.init_value,
+                        'type': state_iterator.state_type,
+                        'timestamp': state_iterator.timestamp,
+                        'meta':
+                        {
+                            'id': state_iterator.uuid,
+                            'type': 'state',
+                            'version': '2.0',
+                            "contract": []
```
Comment (AndreasBomholtz) [inline wappsto/object_instantiation/instantiate.py]: Please remove "contract".

File: wappsto/object_instantiation/instantiate.py [hunk 2]
```diff
@@ -258,3 +258,99 @@ def build_network(self, decoded):
...
+            device = {
+                'name': device_iterator.name,
+                'product': device_iterator.product,
+                'protocol': device_iterator.protocol,
+                'serial': device_iterator.serial_number,
+                'manufacturer': device_iterator.manufacturer,
+                'communication': device_iterator.communication,
+                'description': device_iterator.description,
+                'value': values,
+                'version': '',
```
Comment (AndreasBomholtz) [inline wappsto/object_instantiation/instantiate.py]: Why is this version always empty?

File: wappsto/object_instantiation/instantiate.py [hunk 3]
```diff
@@ -258,3 +258,99 @@ def build_network(self, decoded):
...
+                'meta':
+                {
+                    'id': device_iterator.uuid,
+                    'version': device_iterator.version,
```
Comment (AndreasBomholtz) [inline wappsto/object_instantiation/instantiate.py]: I think this should always be '2.0' for now.

File: test/wappsto_test.py [hunk 4]
```diff
@@ -85,6 +85,17 @@ def create_response(self, verb, callback_exists, trace_id):
     return '{"jsonrpc": "2.0", "id": "1", "params": {"url": "'+url+'",'+trace+' "data": {"meta": {"id": "'+id+'"}, "data": "93"}}, "method": "'+verb+'"}'
 
 
+def get_expected_json(self):
```
Comment (AndreasBomholtz) [inline test/wappsto_test.py]: What are you doing in this function?

File: wappsto/object_instantiation/instantiate.py (hunk 2 above)
Comment (bubriks) [reply wappsto/object_instantiation/instantiate.py]: it was like that in the original JSON so I kept it (but now I changed it to 2.0)

File: test/wappsto_test.py (hunk 4 above)
Comment (bubriks) [reply test/wappsto_test.py]: it gets the JSON file and reads its information, after that it modifies data so it stores the same information as is being expected to be sent

**Pull request merged**